ssas/service/public/tokens.go

```diff
@@ -3,9 +3,7 @@ package public
 import (
 	"fmt"
 	"time"
-
 	"github.com/dgrijalva/jwt-go"
-
 	"github.com/CMSgov/bcda-ssas-app/ssas"
 	"github.com/CMSgov/bcda-ssas-app/ssas/cfg"
 	"github.com/CMSgov/bcda-ssas-app/ssas/service"
```

Comment:
did `gofmt` remove these lines? i thought the imports were supposed to be grouped like this.

Reply:
it was go formated